diff -up libkdegames-20.03.80/colorproxy_p.cpp.omv~ libkdegames-20.03.80/colorproxy_p.cpp
--- libkdegames-20.03.80/colorproxy_p.cpp.omv~	2020-03-23 16:31:57.722734227 +0100
+++ libkdegames-20.03.80/colorproxy_p.cpp	2020-03-23 16:32:06.187724018 +0100
@@ -17,6 +17,7 @@
  ***************************************************************************/
 
 #include "colorproxy_p.h"
+#include <QPainterPath>
 
 //BEGIN QPaintDeviceColorProxy
 
